roles/install-deps/tasks/install-mongodb.yml

```diff
@@ -86,8 +86,8 @@
     - mongodb
     - deps
 
-- name: Install MongoDB
-  when: ansible_distribution == "Debian" and ansible_distribution_major_version == "8"
+- name: Install MongoDB on Stretch
```

Comment:
I thought we were on Buster i.e. Debian 10?

Reply:
yes, we were. We already have entry to install mongo db on [Buster,](https://github.com/kernelci/kernelci-backend-config/blob/master/roles/install-deps/tasks/install-mongodb.yml#L61) it was only necessary update the repo and let this entry to install stretch.

Reply:
OK, but then it's the default package from Debian Stretch which I'm not sure will actually work.

That's fine for now as the initial requirement is to have it working with Mongo v4.2 on Buster, if it works or not on other distros isn't too important.  But as a follow-up, we should decide which distro version we actually support and either test them or drop them.  In doubt, we can just drop everything except Buster.

@khilman Which distro are you using in the Docker image for kernelci-backend?

Reply:
ok, I will update to have only buster as option.

Reply:
@gctucker I removed all that are not related to buster, let me know if I missed anything.